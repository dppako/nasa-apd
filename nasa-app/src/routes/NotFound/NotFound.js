import React from 'react';

const NotFound = () => {
  return (
    <h2 className="siimple-h2">Oops, page not found...</h2>
  );
};

export default NotFound;