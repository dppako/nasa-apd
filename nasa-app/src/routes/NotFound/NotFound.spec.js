import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

describe('NotFound Component', () => {
  it('matches snapshot', () => {
    const component = renderer.create(
      <NotFound />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});