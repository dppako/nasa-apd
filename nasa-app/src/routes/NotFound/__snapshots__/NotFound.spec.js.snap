// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component matches snapshot 1`] = `
<h2
  className="siimple-h2"
>
  Oops, page not found...
</h2>
`;
